Runtime/Core/Context/Context.cs

```diff
@@ -59,22 +63,26 @@ private void GetTriggerableInterfaces()
 
         private void Update()
         {
+            if (!_initialized) return;
             _updatables.ForEach(x => x.Update());
```

Comment:
@ShedsUnlimited consider using the ? operator instead of the _initialized check
```suggestion
            _updatables?.ForEach(x => x.Update());
```